ledger/simulation/tracer.go

```diff
@@ -138,9 +138,6 @@ func (tracer *evalTracer) BeforeTxnGroup(ep *logic.EvalParams) {
 	if ep.PooledApplicationBudget != nil && tracer.result.TxnGroups[0].AppBudgetAdded == 0 {
 		tracer.result.TxnGroups[0].AppBudgetAdded = uint64(*ep.PooledApplicationBudget)
 	}
-	if ep.FeeCredit != nil {
-		tracer.result.TxnGroups[0].FeeCredit = *ep.FeeCredit
-	}
```

Comment:
Seems like the intent was to report the left over credit. Were we just never serializing it?

Reply:
Yes, we never ended up reporting this field in the REST API. I've chosen to remove this for now to clean up the code, but in the future we may want to add this back properly

More context in this thread: https://github.com/algorand/go-algorand/pull/5335#discussion_r1178104933